components/src/atoms/Slider/slider.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { Flex, SPACING } from '@opentrons/components'
+
+import { Slider } from './index'
+
+import type { Meta, StoryObj } from '@storybook/react'
+
+const meta: Meta<typeof Slider> = {
+  title: 'Helix/Atoms/Slider',
+  component: Slider,
+  decorators: [
+    Story => (
+      <Flex
+        marginTop={SPACING.spacing16}
+        width="20rem"
+        height="22rem"
+        justifyContent="center"
+        alignItems="center"
```

Comment:
if you prefer to use `Flex` instead `div`

```suggestion
        justifyContent={JUSTIFY_CENTER}
        alignItems={ALIGN_CENTER}
```